openhands/controller/agent_controller.py

```diff
@@ -720,6 +722,13 @@ async def _step(self) -> None:
         log_level = 'info' if LOG_ALL_EVENTS else 'debug'
         self.log(log_level, str(action), extra={'msg_type': 'ACTION'})
 
+    def _notify_on_llm_retry(self, retries: int, max: int) -> None:
+        if self.status_callback is not None:
+            msg_id = 'STATUS$LLM_RETRY'
+            self.status_callback(
+                'info', msg_id, f'Retrying LLM request, {retries} / {max}'
```

Comment:
This might be better as `warn` rather than `info` if that were a status, but it would need to be added to `frontend/src/services/actions.ts` and that only makes sense if we have others.

Reply:
Agreed! We can leave it for now